app/common/src/map/HotelSnap.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, View, Text, Image } from 'react-native';
+import Color from './../Color';
+
+type Props = {|
+  width: number,
+  hotel: {|
+    name: string,
+  |},
+|};
+
+const styles = StyleSheet.create({
+  description: {
+    flexDirection: 'row',
+  },
+  image: {
+    width: 50,
+    height: 50,
+    borderRadius: 5,
+    marginRight: 10,
+  },
+  hotelName: {
+    fontSize: 18,
+  },
+  metainfo: {
+    color: Color.grey._700,
+    opacity: 0.8,
+  },
+  price: {
+    color: Color.brand,
+    fontWeight: 'bold',
+  },
+});
+
+const HotelSnap = ({ width, hotel }: Props) => (
```

Comment:
Can we find a better name for this? Other than "snap".

Reply:
fixed (hopefully)